modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: REFERENCE
+[id="oc-mirror-image-set-examples_{context}"]
+= Image set configuration examples
+
+Review the following examples of `ImageSetConfiguration` files for various use cases.
+
+[discrete]
+[id="oc-mirror-image-set-examples-helm_{context}"]
+== Include arbitrary images and helm charts
+
+The following `ImageSetConfiguration` file includes a UBI image and helm charts.
+// TODO: get more specific?
+
+.Example `ImageSetConfiguration` file with helm charts
+[source,yaml]
+----
+apiVersion: mirror.openshift.io/v1alpha1
+kind: ImageSetConfiguration
+archiveSize: 4
+mirror:
+ ocp:
+   channels:
+     - name: stable-4.9
+ operators:
+   - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.9
+ helm:
+   repos:
+     - name: redhat-helm-charts
+       url: https://raw.githubusercontent.com/redhat-developer/redhat-helm-charts/master
+       charts:
+         - name: ibm-mongodb-enterprise-helm
+           version: 0.2.0
+ additionalImages:
+   - name: registry.redhat.io/ubi8/ubi:latest
+storageConfig:
+ registry:
+   imageURL: example.com/example/oc-mirror
+   skipTLS: false
+----
+// TODO: Update/strip example as needed
+
+// TODO: Add another example
```

Comment:
@afflom @jpower432 Can you help me with another good example image set config file that we can show?

I updated the main image set config creation procedure to have a pretty stripped down example: https://deploy-preview-40474--osdocs.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#oc-mirror-creating-image-set-config_installing-mirroring-disconnected

And then the first example here that I have adds the helm charts and additional images: https://deploy-preview-40474--osdocs.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#oc-mirror-image-set-examples_installing-mirroring-disconnected

Most of the examples in the repo (https://github.com/openshift/oc-mirror/tree/main/docs/examples) don't have a storage back end in the config. So I could use help with what would be a good second example to show.

Reply:
Thanks @jpower432. Does it matter where the storageConfig is located? In the above example it's at the bottom, so I'd like to know which is the best way and be consistent

Reply:
@bergerhoffer I prefer to keep it at the top because of how long the mirror spec can get, but it will work either way.

Reply:
Thanks, I'll move the other example to have it at the top, for consistency